packages/react/src/components/Forms/Select/index.tsx

```diff
@@ -1,11 +1,27 @@
-import React, { forwardRef, PropsWithChildren } from "react";
+import React, { forwardRef, PropsWithChildren, ReactNode } from "react";
 import { SelectMulti } from ":/components/Forms/Select/multi";
-import { Option, SelectMono } from ":/components/Forms/Select/mono";
+import { SelectMono } from ":/components/Forms/Select/mono";
 import { FieldProps } from ":/components/Forms/Field";
 
 export * from ":/components/Forms/Select/mono";
 export * from ":/components/Forms/Select/multi";
 
+export type OptionWithRender = {
+  disabled?: boolean;
+  value: string;
+  label: string;
+  render: () => ReactNode;
+};
```

Comment:
It is more challenging, but you can search a string inside a `ReactNode`, `value` and `label` would not have to be mandatory.

Reply:
The client has to manage 3 arrays to work with this feature, when only one could be necessary, but it's ok, it is just an observation. 
Why render is a function?

Reply:
- `value` for the data specific format "🇫🇷 Français" -> `french` 
- `label` for the searchable mode "🇫🇷 Français" -> "Français" ( as we can't fit ReactNode inside a `<input>` ) 
- `ReactNode` is obvious



Render is a function because in the future we might we to give context informations maybe, such as disabled, highlighted etc ... this PR is a first-intention at this point.